### ISSUE TITLE
[intro-to-pytorch]Part 3 - Training Neural Networks - autograd

### ISSUE THREAD

**Issue by HaelChan** [udacity/deep-learning-v2-pytorch#36]
In PyTorch [v0.4.1](
https://github.com/pytorch/pytorch/releases/tag/v0.4.1), tensor printing now includes `requires_grad` and `grad_fn` information. Therefore, we don't need to `print(y.grad_fn)` any more.

**Comment by mcleonard** [comment]
Thanks for your feedback @HaelChan. As was pointed out, for v.0.4.0 you need to print the gradient function intentionally. We're sticking with v0.4 until v1.0 is stable, so leaving in that bit of code.
 Thanks!
